website/docs/en/config/experiments.mdx

```diff
@@ -145,116 +144,46 @@ export default {
 In addition, you can also configure a `test` parameter for more fine-grained control over which modules are lazily compiled. The `test` parameter can be a regular expression that matches only those modules that should be lazily compiled. It can also be a function where the input is of type 'Module' and returns a boolean value indicating whether it meets the criteria for lazy compilation logic.
 
 :::tip
-The current lazy compilation aligns with the webpack implementation, **and is still in the experimental stage**. In some scenarios, lazy compilation might not work as expected, or the performance improvement may be insignificant.
+The current lazy compilation **is still in the experimental stage**. In some scenarios, lazy compilation might not work as expected, or the performance improvement may be insignificant.
```

Comment:
We should add other tip to link to "guide/features/lazy-compilation.mdx
" here and explain that this option requires integrating middleware if using a custom server.